qutebrowser/browser/qutescheme.py

```diff
@@ -135,6 +135,11 @@ def data_for_url(url):
     Return:
         A (mimetype, data) tuple.
     """
+    # normalize path and strip redundant trailing slashes
+    norm_url = url.adjusted(QUrl.NormalizePathSegments | QUrl.StripTrailingSlash)
```

Comment:
Please break this line after the `|` so it's not longer than 79 chars.

Reply:
ok